source/isaaclab/isaaclab/managers/manager_term_cfg.py

```diff
@@ -201,12 +201,21 @@ class ObservationGroupCfg:
     concatenate_terms: bool = True
     """Whether to concatenate the observation terms in the group. Defaults to True.
 
-    If true, the observation terms in the group are concatenated along the last dimension.
+    If true, the observation terms in the group are concatenated along the dimension specified in concatenate_dim.
     Otherwise, they are kept separate and returned as a dictionary.
 
     If the observation group contains terms of different dimensions, it must be set to False.
     """
 
+    concatenate_dim: int = -1
+    """Dimension along to concatenate the different oberservation terms. Defaults to -1.
+
+    If concatenate_terms is True, this specifies the dimension along which the observation terms are concatenated.
+    The indicated dimension is the one of the observations, i.e. for a 2D RGB image (H, W, C), the dimension
+    0 means concatenating along the height, 1 along the width and 2 along the channels. The offset due
+    to the batched environment is handled automatically.
```

Comment:
I find this rather confusing honestly. Why not let the user think in terms of batches and make it clear to what it actually is? So `1` means the first index of (B, C, H, W) which is channels.

```suggestion
    If :attr:`concatenate_terms` is True, this parameter specifies the dimension along which the observation terms are concatenated.
    The indicated dimension depends on the shape of the observations. For instance, for a 2D RGB image of shape (H, W, C), the dimension
    0 means concatenating along the height, 1 along the width, and 2 along the channels. The offset due
    to the batched environment is handled automatically.
```